pkg/controller/status.go

```diff
@@ -14,9 +14,14 @@ import (
 )
 
 // ClusterStatus provides status of the cluster
-func (c *Controller) ClusterStatus(team, cluster string) (*spec.ClusterStatus, error) {
+func (c *Controller) ClusterStatus(team, namespace, cluster string) (*spec.ClusterStatus, error) {
+
+	if namespace == "" {
```

Comment:
Drop that (here and in other similar places), as we now require the namespace to be present by the caller